Source/effects.h

```diff
@@ -116,7 +116,8 @@ enum class HeroSpeech {
 	MaybeItsLockedFromTheInside,
 	LooksLikeItsRustedShut,
 	MaybeTheresAnotherWay,
-	LAST = MaybeTheresAnotherWay
+	Death,

```

Comment:
One issue here, all enums here are verbatim the content of the audio, and the player does not say "death" when calling this one 🙂


Preferably transcribe the death sound the warrior makes here.

Reply:
I had considered that. For some reason, I was thinking that it wasn't applicable to this sound effect since the sound clip doesn't match up between the classes, but it occurs to me now that these are all based on the Warrior sound clips anyway.